site/src/component/GradeDist/Chart.tsx

```diff
@@ -131,16 +131,19 @@ export default class Chart extends React.Component<ChartProps> {
    * @return a JSX block rendering the chart
    */
   render() {
+    const data = this.getClassData()
+    const largeGraphScale = data.some((grade) => Object.values(grade).some((value) => value > 999));
```

Comment:
I assume that it's always the case that 1000 will never appear on the scale unless one of the bars has a height of 1000 or more, is that correct?

Also small suggestion but this condition can be rewritten:

```suggestion
    const largeGraphScale = data.some((grade) => grade[grade.id] > 999));
```

Reply:
I'm currently stuck on mobile, but I'd be happy to try and hunt down a class with ~1000 later to check and make sure this is absolutely true.

From my understanding, unless the number of students is greater than the next tick, whatever increment it may be, the label wouldn't display. 

TLDR: Yes, I think you're right about 1,000 not being displayed unless we've got a value >= 1000, but I'm happy to absolutely confirm this later.

Reply:
<img width="392" alt="Screenshot 2023-07-27 at 3 53 14 PM" src="https://github.com/icssc/peterportal-client/assets/100006999/b0668c94-598a-4fb3-9b7e-e145992fa338">

<img width="392" alt="Screenshot 2023-07-27 at 3 52 56 PM" src="https://github.com/icssc/peterportal-client/assets/100006999/57739537-fc38-4738-aaa4-9f5f5ef60fc2">

It's a bit scuffed, but just looking at these two examples and just how other graphs handle new ticks, I'm pretty confident that the current implementation with `> 999` should work! @js0mmer